stratisd_cert.py

```diff
@@ -200,6 +202,34 @@ def setUp(self):
         time.sleep(1)
         exec_command(["udevadm", "settle"])
 
+        if StratisCertify.monitor_dbus is True:
+            # pylint: disable=consider-using-with
+            self.trace = subprocess.Popen(
+                [
+                    MONITOR_DBUS_SIGNALS,
+                    "org.storage.stratis3",
+                    "/org/storage/stratis3",
+                ],
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                shell=False,
+            )
+
+    def tearDown(self):
+        """
+        Tear down an individual test.  For now, this only stops the
+        D-Bus trace.
+        :return: None
+        """
+        trace = getattr(self, "trace", None)
+        if trace is not None:
+            time.sleep(1)
+            self.trace.send_signal(signal.SIGINT)
+            (_, stderrdata) = self.trace.communicate()
+            self.trace.wait(timeout=1)
```

Comment:
The wait command must be unnecessary. If communicate returned, the process has terminated. Note that communicate() takes a timeout argument as well, if you want one.